.github/workflows/publish-chart.yml

```diff
@@ -0,0 +1,53 @@
+name: Build and Publish Helm Chart
+
+permissions:
+  contents: read
+
+on:
+  release:
+    types:
+    - published
+  push:
+    branches:
+    - main
+    tags:
+    - v*.*.*
+  pull_request:
+    branches:
+    - main
+    paths-ignore:
+    - 'docs/**'
+    - '**/*.md'
+    types: [labeled, unlabeled, opened, synchronize, reopened]
+
+jobs:
+  publish-charts:
+    runs-on: ubuntu-latest
+    if: github.event_name != 'pull_request' || contains(github.event.pull_request.labels.*.name, 'ok-to-chart')
+    steps:
+    - name: Check out repository
+      uses: actions/checkout@v4
+    - name: Set up Helm
+      uses: azure/setup-helm@v4.3.0
+      with:
+        version: v3.16.2
+    - name: Configure Git
+      run: |
+        git config user.name "$GITHUB_ACTOR"
+        git config user.email "$GITHUB_ACTOR@users.noreply.github.com"
+    - name: Set charts version
+      if: github.event_name == 'push'
+      run: |
+        sed -i "s/version: .*/version: ${GITHUB_REF_NAME#v}/" dist/chart/Chart.yaml
+    - name: Move crds folder out of templates
+      run: mv dist/chart/templates/crd dist/chart/crds
+    - name: Run chart-releaser
+      uses: afritzler/helm-oci-charts-releaser@enh/skip-gh-release
+#     uses: bitdeps/helm-oci-charts-releaser@v0.1.3
```

Comment:
We can now use the official 0.1.4 version as this has the `skip_gh_release` feature in.